plugin/vcs/gitlab/gitlab.go

```diff
@@ -272,11 +278,17 @@ func (provider *Provider) FetchRepositoryActiveMemberList(ctx context.Context, o
 	// we only return active member (both state and membership_state is active)
 	activeRepositoryMember := make([]*vcs.RepositoryMember, 0)
 	for _, gitLabMember := range gitLabrepositoryMember {
-		if gitLabMember.State == vcs.StateActive && gitLabMember.MembershipState == vcs.StateActive {
+		if gitLabMember.State == vcs.StateActive {
+			// the Email filed cannot be returned as expected, maybe cause by: https://gitlab.com/gitlab-org/gitlab/-/issues/25077
+			// so we need to fetch this field one by one
+			userInfo, err := provider.FetchUserInfo(ctx, oauthCtx, instanceURL, &gitLabMember.ID)
+			if err != nil {
+				return nil, err
+			}
 			gitLabRole, bytebaseRole := getRoleAndMappedRole(gitLabMember.AccessLevel)
 			repositoryMember := &vcs.RepositoryMember{
 				Name:         gitLabMember.Name,
-				Email:        gitLabMember.Email,
+				Email:        userInfo.Email, /* this filed should has been returned by projects/%s/members/all */
```

Comment:
The comment above provider.FetchUserInfo is good enough. No need to mention here